docs/documentation/development/bindings/thing-definition.md

```diff
@@ -103,7 +102,9 @@ The following XML snippet shows a system channel-type definition and thing-type
 </channel-type>
 ```
 
-There exist systemwide channels that are available by default:
+### Default system-wide channel types
```

Comment:
Ah, sub-headers, excellent! I planned to add them since weeks, because I was never able to point to those sections!

Now you should only capitalise them and all is fine :-) And I would maybe shorten it to "System State Channel Types"